src/main/kotlin/blackjack/view/ResultView.kt

```diff
@@ -0,0 +1,104 @@
+package blackjack.view
+
+import blackjack.domain.BlackjackResult
+import blackjack.domain.Card
+import blackjack.domain.CardNumber
+import blackjack.domain.CardShape
+import blackjack.domain.Dealer
+import blackjack.domain.Dealer.Companion.HIT_STANDARD_SCORE
+import blackjack.domain.Participant
+import blackjack.domain.Participant.Companion.INIT_CARD_SIZE
+import blackjack.domain.Player
+import blackjack.domain.ResultType
+
+object ResultView {
+    private const val SET_UP_MESSAGE = "\n%s와 %s에게 $INIT_CARD_SIZE 장의 카드를 나누었습니다."
```

Comment:
```suggestion
    private const val SET_UP_MESSAGE = "\n%s와 %s에게 ${Participant.INIT_CARD_SIZE} 장의 카드를 나누었습니다."
```
companion object를 import하게되면 간편하게 사용할 수 있는 장점이 있습니다.
반면, 이 값이 어느 곳에서 정의가 되었는지 알기가 어렵습니다.
어떤 companion object에 정의가 되어있는지 알기 쉽도록 코드를 개선해보면 어떨까요? (이하 관련 내용 동일)